package/features/steps/cli_steps.py

```diff
@@ -161,7 +161,10 @@ def _check_kedroviz_running(context):
     data_json = json.loads(download_url("http://localhost:4141/api/nodes.json"))
     try:
         assert context.result.poll() is None
-        assert "predict" in data_json["nodes"][0]["full_name"]
+        assert (
```

Comment:
The reason we have to sort this now is that with Kedro 0.15 and 0.16, `pipelines` returns 
```
{
      "de": data_engineering_pipeline,
      "ds": data_science_pipeline,
      "__default__": data_engineering_pipeline + data_science_pipeline,
 }
```
so, it will get `nodes` from `de` first, and skip the duplicate from `__default__`.

Reply:
It's actually getting `de` first, and then processes `__default__`, because the dictionary we are passing is 
```
{"de": de_pipeline, 
 "ds": ds_pipeline,
 "__default__": de_pipeline + ds_pipeline
}
```
and not 
```
{
 "__default__": de_pipeline + ds_pipeline
 "de": de_pipeline, 
 "ds": ds_pipeline
}
```

Processing `__default__` will skip the duplicate from the one `de`, where some of the nodes were already processed.  

It's not a problem, the reason I have to sort it now is that the previous checking 
`"predict" in data_json["nodes"][0]["full_name"]` always assumed that `predict` is in the first `nodes`, which is not the case with multiple pipelines.